### PR TITLE
Add support for precompiled assets

### LINKED ISSUE #52: Saving page results in Internal Server Error
I Created a new page, set title, parent and added some content.

When clicking save I got this error:
```
ActionView::Template::Error (undefined method `find_asset' for nil:NilClass):
     5:     <meta charset='<%= ::Rails.application.config.encoding %>' />
     6:     <meta httpequiv="XUACompatible" content="IE=edge,chrome=1" />
     7:     <%= stylesheet_link_tag "wymeditor/skins/refinery/wymiframe", :media => "all" %>
     8:     <%= stylesheet_link_tag "formatting", :media => "all" if Rails.application.assets.find_asset('formatting.css') %>
     9:     <%= stylesheet_link_tag "theme", :media => "all" %>
    10:     <%= javascript_include_tag 'modernizr-min' %>
    11:   </head>
```

I am using (pre-)compiled assets on my production environment.

The error seems to be related to https://github.com/rails/sprockets-rails/issues/311

### PR DESCRIPTION
**Pull request by veger**
Should fix #52 (at least it does for my application :wink: )

I could not run the tests locally, so hopefully they'll pass

### CODE REVIEW

**Comment by veger** [comment]
build is failing due to some dependency issue...?

**Comment by parndt** [comment]
@veger can you please pull the latest master into your branch?

**Comment by veger** [comment]
I rebased my PR on master

**Comment by veger** [comment]
Test fail because the introduced helper is not included in the test application.
TBH I have no idea how the engines work exactly and how to fix this.

Hopefully you know how to add it (to the dummy app?) so the test can pass?

**Comment by parndt** [comment]
@veger gotcha - thanks! I'll see if I can figure it out for you

File: app/views/refinery/wymiframe.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta charset='<%= ::Rails.application.config.encoding %>' />
     <meta httpequiv="XUACompatible" content="IE=edge,chrome=1" />
     <%= stylesheet_link_tag "wymeditor/skins/refinery/wymiframe", :media => "all" %>
-    <%= stylesheet_link_tag "formatting", :media => "all" if Rails.application.assets.find_asset('formatting.css') %>
+    <%= stylesheet_link_tag "formatting", :media => "all" if AssetHelper.asset_exists? 'formatting.css' %>
```
Comment (parndt) [inline app/views/refinery/wymiframe.html.erb:8]:
```suggestion
    <%= stylesheet_link_tag "formatting", :media => "all" if AssetHelper.asset_exists?('formatting.css') %>
```

**Comment by parndt** [comment]
well the UI let me merge this with 0 checks passed, so hopefully they pass.

**Comment by parndt** [comment]
> well the UI let me merge this with 0 checks passed, so hopefully they pass.

they did not 😭

**Comment by parndt** [comment]
Fixed my early merge with #64

**Comment by veger** [comment]
It took a while, but it is nice to see this PR got merged :smile: :tada:

**Pull request merged**